test/structure.js

```diff
@@ -1,61 +1,59 @@
 'use strict';
 
+const metatests = require('metatests');
 const metaschema = require('..');
 
 metaschema.load('geometry', (err, schema) => {
```

Comment:
`metatests` don't support such usage and this may lead to multiple 'results' printed in the end.
All tests should be added synchronously. Therefore this must be changed to smth like
```js
const geometryTest = metatests.test('geometry');
metaschema.load('geometry', (err, schema) => {
  // ... use geometryTest and/or metaschema.case ...
  // ...
  geometryTest.end();
})
```

Or
```js
metatests.test('geometry', (test) => {
  metaschema.load('geometry', (err, schema) => {
    // ... use test and/or metaschema.case ...
    // ...
    geometryTest.end();
  })
});
```

Whichever you prefer.